CWE/events/italy.txt

```diff
@@ -54,460 +126,341 @@ country_event = {
+		set_country_flag = ITA_Togliatti_shooting
+		ai_chance = { factor = 0.9 }
 	}
 
-	mean_time_to_happen = { months = 24 }
-
 	option = {
-		name = EVT_8007740_A
-		any_pop = { militancy = 2.0 consciousness = 5 }
-		relation = { who = USA value = 100 } 
-		country_event = 8007742
+		name = "Treat the shooter lightly"
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-	}
-	
-	option = {
-		name = "Let the ICP remain legal"
+                scaled_militancy = { factor = 7 ideology = communist }
+			}	
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
-	}
-	
-}
-
-#Radicalization of Italian Communist Party
-country_event = {
-  id = 8007741
-  title = EVT_8007741_NAME
-  desc = EVT_8007741_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes
-
-	option = {
-		name = EVT_8007741_A
-		any_pop = { militancy = 1.0 consciousness = 1 }
-		set_country_flag  = very_radical_ICP
+                scaled_militancy = { factor = 7 ideology = communist_social }
+			}	
+        any_pop = {
+                scaled_militancy = { factor = 7 ideology = socialist }
+			}		
+        any_pop = {
+                scaled_consciousness = { factor = 4 ideology = communist }
+			}	
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.02 value = communist }
-		}
-	}
-  
-}
-
-#Dispersion of Italian Communist Party
-country_event = {
-  id = 8007742
-  title = EVT_8007742_NAME
-  desc = EVT_8007742_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes
-
-	option = {
-		name = EVT_8007742_A
+                scaled_consciousness = { factor = 4 ideology = communist_social }
+			}	
+        any_pop = {
+                scaled_consciousness = { factor = 4 ideology = socialist }
+			}
 		any_pop = {
 			limit = { is_primary_culture = yes }
-			ideology = { factor = -0.10 value = communist }
+			ideology = { factor = 0.015 value = communist }
 		}
+		add_country_modifier = { name = revolution_chaos duration = 365 }
+		set_country_flag = ITA_Togliatti_shooting
+		ai_chance = { factor = 0.1 }
 	}
 	
 }
 
-#Italian Communist Party at helm
+#The Special Laws
 country_event = {
-  id = 8007743
-  title = EVT_8007743_NAME
-  desc = EVT_8007743_DESC
-  picture = "nwo2_italian_communist_party"
+	id = 8007772
+	title = "The $YEAR$ Special Laws"
+	desc = EVTDESC8007772
+	picture = "italian_parliament"
   
 	fire_only_once = yes
-  
+
 	trigger = {
 		tag = ITA
-		has_global_flag = blocsenabled
-		ruling_party_ideology = communist
-		government = proletarian_dictatorship
-		election = no
-	}
-
-	option = {
-		name = EVT_8007743_A
-		
-		#Break alliances with non communists
-		any_country = {
-			limit = { NOT = { ruling_party_ideology = communist } }
-			leave_alliance = THIS
-			relation = { who = THIS value = -200 }
-		}
-		
-		#Be friends with fellow communists
```

Comment:
This should also be retained since fellow communists would be pleased with this development